cmd/experimental/kjobctl/apis/v1alpha1/application_profile_types.go

```diff
@@ -121,15 +120,16 @@ type SupportedMode struct {
 	// The raycluster flag used only for the RayJob mode.
 	// The request flag used only for Interactive and Job modes.
 	// The cmd flag used only for Interactive, Job, and RayJob.
+	// The skip-priority-workload and priority flags can be used in all modes.
 	// If the raycluster flag are set, none of localqueue, replicas, min-replicas, or max-replicas can be set.
 	// For the Slurm mode, the possible values are: array, cpus-per-task, error, gpus-per-task, input, job-name, mem, mem-per-cpu,
-	// mem-per-gpu, mem-per-task, nodes, ntasks, output, partition, localqueue, priority.
+	// mem-per-gpu, mem-per-task, nodes, ntasks, output, partition, localqueue.
 	//
 	// cmd and requests values are going to be added only to the first primary container.
 	//
 	// +optional
 	// +listType=set
-	// +kubebuilder:validation:MaxItems=13
+	// +kubebuilder:validation:MaxItems=12
```

Comment:
Not sure that we need to decrease it.

Reply:
We remove here only validation rules, which indeed was removed

Reply:
Yeah, but count of Items shouldn't be changed. No?

Reply:
agree